conans/client/build/visual_environment.py

```diff
@@ -3,7 +3,8 @@
 
 from conans.client.build.compiler_flags import build_type_define, build_type_flags, format_defines, \
     include_path_option, parallel_compiler_cl_flag, visual_runtime
-from conans.client.build.cppstd_flags import cppstd_flag, cppstd_from_settings
+from conans.client.build.cppstd_flags import cppstd_from_settings
+from conans.tools import cppstd_flag
```

Comment:
same as commented by @memsharded. In internal imports

Reply:
changed